x-pack/plugins/secops/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "author": "Elastic",
+  "name": "sec-ops",
+  "version": "7.0.0-alpha1",
+  "scripts": {
+    "build-graphql-types": "node scripts/generate_types_from_graphql.js"
+  },
+  "devDependencies": {
+    "@types/boom": "7.2.0",
+    "@types/color": "^3.0.0",
+    "@types/lodash": "^4.14.110"
+  },
+  "dependencies": {
+    "boom": "3.1.1",
```

Comment:
This seems outdated and something bad will happen to all living creatures within this repo as within the main package.json they have this version:

```
"boom": "^7.2.0",
```

https://github.com/elastic/kibana/blob/master/package.json#L96